docs/development.md

```diff
@@ -11,35 +11,34 @@
 
```

Comment:
Table of Contents should have "Developing with Tilt". Please run doctoc.

Reply:
fixed